libsolidity/interface/StandardCompiler.cpp

```diff
@@ -131,6 +132,11 @@ Json::Value formatErrorWithException(
 		formatSourceLocation(boost::get_error_info<errinfo_sourceLocation>(_exception)),
 		formatSecondarySourceLocation(boost::get_error_info<errinfo_secondarySourceLocation>(_exception))
 	);
+
+	if (_errorId)
+		error["errorCode"] = static_cast<unsigned>(_errorId.value().error);
```

Comment:
Do we really want this to be an integer instead of a string?

Reply:
Yes, string would be more flexible. And one casting less.